public/javascripts/SVValidate/src/keyboard/Keyboard.js

```diff
@@ -82,10 +82,55 @@ function Keyboard(menuUI) {
         }
     }
 
+    function _setSeverity(severity) {
+        console.log(severity);
+        if(svv.newValidateBeta){
+            svv.ui.newValidateBeta.severityMenu.find(`#severity-button-${severity}`).click();
+        } else {
+            menuUI.severityMenu.find('.severity-level').removeClass('selected');
+        }
+    }
+
+    function _setDisagreeReason(reasonNumber) {
+        if(svv.newValidateBeta){   
+            svv.ui.newValidateBeta.disagreeReasonOptions.find(`#no-button-${reasonNumber}`).click();
+        } else {
+            menuUI.disagreeReasonOptions.find('.validation-reason-button').removeClass('chosen');
+        }
+    }
+
+    function _setUnsureReason(reasonNumber) {
+        if(svv.newValidateBeta){    
+            svv.ui.newValidateBeta.unsureReasonOptions.find(`#unsure-button-${reasonNumber}`).click();
+        } else {
+            menuUI.unsureReasonOptions.find('.validation-reason-button').removeClass('chosen');
+        }
+    }
+
+    function handleEscapeKey() {
+        if (status.addingComment) {
+            document.activeElement.blur();
+            status.addingComment = false;
+        } else if (menuUI.noButton.hasClass('chosen') && svv.newValidateBeta) {
```

Comment:
What's the goal for the cases after the first one? Isn't `status.addingComment` set to `true` when you're typing in any of the text boxes? Won't it always go to the first case, or is there something I'm missing?